MonoGame.Framework/Android/Input/Keyboard.cs

```diff
@@ -120,18 +124,60 @@ internal static void KeyUp(Keycode keyCode)
 			maps[Keycode.W] = Keys.W;
 			maps[Keycode.X] = Keys.X;
             maps[Keycode.Y] = Keys.Y;
-            maps[Keycode.C] = Keys.Z;
-			maps[Keycode.Back] = Keys.Escape;
+            maps[Keycode.Z] = Keys.Z;
+            maps[Keycode.Space] = Keys.Space;
+			maps[Keycode.Escape] = Keys.Escape;
```

Comment:
formatting looks a bit off here? spaces vs tabs?

Reply:
formatting is off in that file in general